src/datastructure/CameraParametersCollection.cpp

```diff
@@ -127,6 +127,12 @@ FrameworkReturnCode CameraParametersCollection::suppressCameraParameters(const u
 	}
 }
 
+void CameraParametersCollection::suppressAllCameraParameters() 
+{
+    m_cameraParameters.clear();
+    m_id = 0;
+}
+
 bool CameraParametersCollection::isExistCameraParameters(const uint32_t id) const
 {
     if (m_cameraParameters.find(id) != m_cameraParameters.end())
```

Comment:
nit: out of scope of this PR, but `return m_cameraParameters.find(id) != m_cameraParameters.end()` would be nicer here.